app/src/main/java/com/example/dine_and_donate/CurrentVouchersFragment.java

```diff
@@ -0,0 +1,52 @@
+package com.example.dine_and_donate;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.recyclerview.widget.StaggeredGridLayoutManager;
+
+import com.example.dine_and_donate.Adapters.StaggeredRecyclerViewAdapter;
+
+import java.util.ArrayList;
+
+public class CurrentVouchersFragment extends Fragment {
```

Comment:
i think this might be a merge conflict issue? this isn't your code right? i think you can delete this...